sema/rocreator/rocreator.yml

```
@@ -0,0 +1,53 @@
+        content-type: text/csv; charset=utf8
+  list:
+    "./specialfile":  # additional individual file
+      field-name: field-value
+    https://whatever:
+      autifill: off
+      field-name: field-value
+  # check if yaml has a built in extensiom  # check if yaml has a built in extensiom
+
+  # extra --> check with Jorge / github docs --> there should be a way to expose generated content on github pages without the need for gh-pages branch!
+  #  see https://github.com/emo-bon/observatory-bpns-crate/settings/pages
+  #  and specially https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-with-a-custom-github-actions-workflow 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)